docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -372,6 +397,22 @@ Oracle                      ``SDO_GEOM.VALIDATE_GEOMETRY_WITH_CONTEXT(poly, 0.05
 *Availability*: `PostGIS <https://postgis.net/docs/ST_Overlaps.html>`__,
 Oracle, MySQL, SpatiaLite, PGRaster (Bilateral)
 
+Tests if the geometry field spatially overlaps the lookup geometry.
+
+==========  =================================================
+Backend     SQL Equivalent
+==========  =================================================
+PostGIS     ``ST_Overlaps(poly, geom)``
+Oracle      ``SDO_OVERLAPS(poly, geom)``
+MySQL       ``ST_Overlaps(poly, geom)``
+SpatiaLite  ``Overlaps(poly, geom)``
+==========  =================================================
```

Comment:
If this table was missing, should it be added as a separate commit first to keep the diff to a minimum?

Reply:
Yes, since that change could be backported.